### DISCUSSION TITLE
Impact on Azure function due to OPENSSL security Advisory

### DISCUSSION THREAD

**Discussion by btharunkumar** [Azure/Azure-Functions#2293] [Polls]
Hello team,

I can see that there is a new release in OPENSSL security Advisory
Reference: https://www.openssl.org/news/secadv/20221101.txt

I would like to know if this release may affect any Azure functions 

Please share your observations so that it will be helpful to take further actions

**Comment by fabiocav** [comment]
Closing this as it is a stale discussion and no longer applicable. If you have any additional questions, please let us know. Thank you!
